vas/task_1/src/app/navigation/navigation.component.html

```diff
@@ -0,0 +1,6 @@
+<nav>
+  <a routerLink="home" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">Home</a>
```

Comment:
```suggestion
  <a routerLink="home" routerLinkActive="active">Home</a>
```

`[routerLinkActiveOptions]="{exact: true}"` is not necessary and could be skipped

Reply:
should we know more abpout that or just this skipped code can have the same functionality regarding thsi specific case. I made the change.